src/extensions/mountainview/MVSortingView/CurationControl.tsx

```diff
@@ -59,18 +74,28 @@ const CurationControl: FunctionComponent<Props & SizeMeProps> = ({ selection, se
     const labels = Object.keys(labelCounts).sort()
     const labelRecords: LabelRecord[] = labels.map(label => ({
         label,
-        partial: labelCounts[label] < selection.selectedUnitIds.length ? true : false
+        partial: labelCounts[label] < selectedUnitIds.length ? true : false
     }))
     const paperStyle: React.CSSProperties = {
         marginTop: 25,
         marginBottom: 25,
         backgroundColor: '#f9f9ff'
     }
-    const enableApply = selection.selectedUnitIds.length > 0
+    const unitsAreInMergeGroups = (unitIds: number[]) => {
+        const mg = curation.mergeGroups || []
+        const all = mg.reduce((prev, g) => [...prev, ...g], []) // all units in merge groups
+        for (let unitId of unitIds) {
+            if (!all.includes(unitId)) return false
+        }
+        return true
+    }
+    const enableApply = selectedUnitIds.length > 0
+    const standardChoices = ['accept', 'reject', 'noise', 'artifact', 'mua']
```

Comment:
Always offer some standard choices for labels, and then also offer custom choices for labels.

Reply:
Nice to see custom labels back!